scripts/utils/create-jest-config.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+module.exports = (resolve, rootDir) => {
+  const config = {
+    automock: false,
```

Comment:
It's been my plan to slowly move away from automocking as a default and towards explicit mocks using `jest.mock('path/to/module')`. This, the react-native-preset and the Jest documentation recommend disabling automocking but I have been too chicken to make this the default.

Let's do it for Jest 15, which should be out in less than a month or so?